### ISSUE TITLE
Installing pybombs starts to play pacman

### ISSUE THREAD

**Issue by TylerTucker** [gnuradio/pybombs#571]
When installing pyBOMBS on Ubuntu, something strange happens...the installer starts to play Pacman. The system I was installing on so happens to have the classic game installed, but I guess the installer tries executing this expecting this to be an Arch Linux system! The installer hangs as its trying to beat my high score - even closing it causes it to reopen again and again.

![image](https://user-images.githubusercontent.com/7915585/73562474-344c8780-4429-11ea-8c41-12e2e7053b41.png)

I used pyBOMBS 2.3.4a0 on Ubuntu 18.04 LTS for reference.

**Comment by Architector4** [comment]
Yeah, the problem seems to be at https://github.com/gnuradio/pybombs/blob/master/pybombs/packagers/pacman.py#L37

Judging by the looks it just checks to see if a program called `pacman` can be run, and, if so, assumes Arch Linux's `pacman` is available and runs it with various commands.

**Comment by mbr0wn** [comment]
This is the most hilarious bug I've ever seen.

**Comment by jkbecker** [comment]
I'm very torn between fixing this and keeping this open forever because it's so amazing. Who doesn't like a surprise game of pacman... :D
